mergify_engine/actions/merge.py

```diff
@@ -70,6 +72,55 @@ def validate_config(self, mergify_config: "rules.MergifyConfig") -> None:
             {"priority": 0, "speculative_checks": 1}
         )
 
+    def _subscription_status(
+        self, ctxt: context.Context
+    ) -> typing.Optional[check_api.Result]:
+
+        if self.config["update_bot_account"] and not ctxt.subscription.has_feature(
+            subscription.Features.MERGE_BOT_ACCOUNT
+        ):
+            return check_api.Result(
+                check_api.Conclusion.ACTION_REQUIRED,
+                "Merge with `update_bot_account` set are unavailable",
```

Comment:
ditto, if you use `are` we need `Merges`
or just go for `is` :)

Reply:
Go for `is` then